### PR TITLE
Update Polish (pl) localization of AMO Frontend 170302

### PR DESCRIPTION
**Pull request by piotrdrag**
@splewako

### CODE REVIEW

File: locale/pl/LC_MESSAGES/amo.po [hunk 1]
```diff
@@ -78,20 +76,19 @@ msgstr "Używane przez"
 
 #: src/amo/components/AddonMoreInfo.js:24
 msgid "Homepage"
-msgstr ""
+msgstr "Strona domowa"
 
 #: src/amo/components/AddonMoreInfo.js:32
 msgid "Support Site"
-msgstr ""
+msgstr "Strona pomocy"
```
Comment (splewako) [inline locale/pl/LC_MESSAGES/amo.po]: "Strona wsparcia" maybe? https://github.com/aviarypl/team/issues/4

File: locale/pl/LC_MESSAGES/amo.po [hunk 2]
```diff
@@ -78,20 +76,19 @@ msgstr "Używane przez"
...
 
 #: src/amo/components/AddonMoreInfo.js:38
 msgid "More information"
 msgstr "Więcej informacji"
 
 #: src/amo/components/AddonMoreInfo.js:42
-#, fuzzy
 msgid "Add-on Links"
-msgstr "Ikona dodatku"
+msgstr "Odnośniki dodatku"
```
Comment (splewako) [inline locale/pl/LC_MESSAGES/amo.po]: Sounds weird and suspicious .

Comment (splewako) [inline locale/pl/LC_MESSAGES/amo.po]: (like wrongly labeled headline/title)

File: locale/pl/LC_MESSAGES/amo.po (hunk 1 above)
Comment (piotrdrag) [inline locale/pl/LC_MESSAGES/amo.po]: The more I think about it, the more I like “Pomoc” to describe a place (as opposed to an action, where “wspierać” is appropriate.)

File: locale/pl/LC_MESSAGES/amo.po (hunk 2 above)
Comment (piotrdrag) [inline locale/pl/LC_MESSAGES/amo.po]: @muffinresearch could you tell us what does it mean?

File: locale/pl/LC_MESSAGES/amo.po (hunk 1 above)
Comment (splewako) [inline locale/pl/LC_MESSAGES/amo.po]: I don't know if I get support as a place idea right but I think we should make it clear in https://github.com/aviarypl/team/issues/4 (and fix here later if needed)

File: locale/pl/LC_MESSAGES/amo.po (hunk 2 above)
Comment (muffinresearch) [inline locale/pl/LC_MESSAGES/amo.po]: The heading covers the add-on's support url and homepage link so it's an umbrella title in this context if that makes sense?

Comment (piotrdrag) [inline locale/pl/LC_MESSAGES/amo.po]: @splewako how about “Więcej informacji”?

Comment (piotrdrag) [inline locale/pl/LC_MESSAGES/amo.po]: I’ve changed it to “Strony dodatku”.

**Comment by muffinresearch** [comment]
Let me know when this is ready to land and I'll merge it.

File: locale/pl/LC_MESSAGES/amo.po (hunk 2 above)
Comment (splewako) [inline locale/pl/LC_MESSAGES/amo.po]: Good idea.

**Comment by piotrdrag** [comment]
It’s ready, thank you.

**Comment by caitmuenster** [comment]
Thanks for your localization contributions, Piotr!

**Pull request merged**